nautobot_plugin_chatops_meraki/worker.py

```diff
@@ -364,7 +365,8 @@ def get_wlan_ssids(dispatcher, org_name=None, net_name=None):
         return prompt_for_organization(dispatcher, "meraki get-wlan-ssids")
     if not net_name:
         return prompt_for_network(dispatcher, f"meraki get-wlan-ssids {org_name}", org_name)
-    ssids = get_meraki_network_ssids(org_name, net_name)
+    client = MerakiClient(api_key=MERAKI_DASHBOARD_API_KEY)
```

Comment:
Should this be a constant at top of file instead of defining multiple times?

Reply:
Do you mean `client`? I am not sure, as it establishes a connection, so I would not put it outside a function or a class.